packages/cli/src/commands/generate/page/templates/page.tsx.template

```diff
@@ -7,6 +8,10 @@ type ${pascalName}PageProps = {
 const ${pascalName}Page = (<%- paramName === '' ? '' : `${propParam}: ${pascalName}PageProps` %>) => {
   return (
     <>
+      <MetaTags
+        title="${pascalName}"
```

Comment:
I don't have strong opinion for that. At the end I think everyone will change the content of the template and just have something relevante to the page to see quickly the difference is enough.

Reply:
> At the end I think everyone will change the content of the template and just have something relevant

Ah, right. This is simply the generator and will give people a head start.

Otherwise that template would have to be stored in a config and .... too complicated for the value.